invenio_accounts/config.py

```diff
@@ -50,6 +50,12 @@
 By default, it uses Celery if it can find it.
 """
 
+ACCOUNTS_SESSION_ACTIVITY_ENABLE = True
```

Comment:
Should we use the past tense (so `ACCOUNTS_SESSION_ACTIVITY_ENABLED`) for this kind of settings? It makes more sense to me and I see it's used in other places as well (when I grep for `_ENABLE` in invenio packages, I usually find `_ENABLED` instead).